### PR TITLE
 Man page updates

### PR DESCRIPTION
**Pull request by agrawalravi90**
#### Bug/feature Description
* Bringing man page updates for latest PBSPro features

#### Checklist:

- [X] I have joined the **[pbspro community forum](http://community.pbspro.org/)**.
- [X] My pull request contains a **single, signed** commit. See **[setting up gpg signature](https://pbspro.atlassian.net/wiki/display/DG/Signing+Your+Git+Commits)**.
- [X] My code follows the **[coding style](https://pbspro.atlassian.net/wiki/display/DG/Coding+Standards)** of this project.
- [ ] My change requires project documentation. See **[required documentation checklist](https://pbspro.atlassian.net/wiki/display/DG/Checklist+for+Developing+Features+and+Bug+Fixes)** for details.
   - [ ] I have added documentation in the **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)**.
- [ ] I have added new **PTL test(s) to my commit**. (See **[using PTL for testing](https://pbspro.atlassian.net/wiki/display/DG/Using+PTL+for+Testing)**) *(or)*
   - [ ] I have added  **manual test(s) to this pull request and explained why PTL is not appropriate** for this case.
- [ ] All new and existing automated tests have passed. (See **[running automated PTL tests](https://pbspro.atlassian.net/wiki/display/DG/PTL+Quick+Start+Guide)**).
- [ ] I have attached **test logs to this pull request** as evidence of testing/verification.

__***For further information please visit the [Developer Guide Home](https://pbspro.atlassian.net/wiki/display/DG/Developer+Guide+Home).***__

### CODE REVIEW

**Comment by arungrover** [comment]
@agrawalravi90 please change makefile.am to remove unnecessary files

File: doc/Makefile.am [hunk 1]
```diff
@@ -54,7 +56,9 @@ notrans_dist_man1_MANS = \
 	man1/qselect.1B \
 	man1/qsig.1B \
 	man1/qstat.1B \
-	man1/qsub.1B
+	man1/qsub.1B \
+	man1/xpbs.1B \
```
Comment (arungrover) [inline doc/Makefile.am]: these two lines are not needed

**Comment by agrawalravi90** [comment]
Thanks for catching that Arun, I've addressed your comment, please let me know if it looks okay now.

**Comment by agrawalravi90** [comment]
Thanks Arun! Requesting one more review please

**Comment by agrawalravi90** [comment]
knock knock, requesting one more review please, thanks!

**Comment by agrawalravi90** [comment]
Thanks Bhroam, I just rebased, will let you guys know once CI passes.

**Pull request merged**